adapters/yeahmobi/yeahmobi.go

```diff
@@ -0,0 +1,179 @@
+package yeahmobi
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"github.com/golang/glog"
+	"github.com/mxmCherry/openrtb"
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/errortypes"
+	"github.com/prebid/prebid-server/macros"
+	"github.com/prebid/prebid-server/openrtb_ext"
+	"net/http"
+	"text/template"
+)
+
+type YeahmobiAdapter struct {
+	EndpointTemplate template.Template
+}
+
+func NewYeahmobiBidder(endpointTemplate string) adapters.Bidder {
+	tpl, err := template.New("endpointTemplate").Parse(endpointTemplate)
+	if err != nil {
+		glog.Fatal("Unknow url template")
+		return nil
+	}
+	return &YeahmobiAdapter{EndpointTemplate: *tpl}
+}
+
+func (adapter *YeahmobiAdapter) MakeRequests(request *openrtb.BidRequest, reqInfo *adapters.ExtraRequestInfo) ([]*adapters.RequestData, []error) {
+	var adapterRequests []*adapters.RequestData
+
+	adapterRequest, errors := adapter.makeRequest(request)
+	if adapterRequest != nil {
```

Comment:
It's more common in Go to check for `errors != nil` to determine if the return variables are safe to use.